src/contracts/interfaces/IStakeRegistry.sol

```diff
@@ -60,6 +60,9 @@ interface IStakeRegistry is IRegistry {
      */
     function deregisterOperator(bytes32 operatorId, bytes memory quorumNumbers) external;
 
+    /// @notice In order to register for a quorum i, an operator must have at least `minimumStakeForQuorum[i]`, as
+    function minimumStakeForQuorum(uint256 quorumNumber) external view returns (uint96);
```

Comment:
comment needs updating / completion; it just kinda trails off right now

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/8888f2a31ff9e13bca8370846c35f1c48b45d48f